bin/build.js

```diff
@@ -0,0 +1,19 @@
+const fs = require( 'fs' ),
+	path = require( 'path' );
```

Comment:
Not a big fan of this kind of `require`ing. Can you separate this into two `require`s?

```js
const fs = require( 'fs' );
const path = require( 'path' );
```